docs/pages/documentation/8-deploy-grakn/grakn-deployment-guide.md

```diff
@@ -19,21 +19,15 @@ This guide offers advice on how to:
 * upgrade to the latest version of the distribution.
    
    
-Grakn releases from our [GitHub repository](https://github.com/graknlabs/grakn) are self-contained packages (tar.gz/zip) containing all the components you need to run Grakn:
-
-* Grakn stack
-* Redis
-* Cassandra
-
-You can run all these components separately depending on the redundancy levels you require.
+Grakn releases from our [GitHub repository](https://github.com/graknlabs/grakn) are self-contained packages (tar.gz/zip) containing all the components you need to run Grakn.
 
 ## Setting up Grakn
 
 ### Minimum Requirements
 
 Grakn can run on default Java settings (heap of 768MB, 1GB machine) if the knowledge base is small enough. Recommended production settings are at least 4GB machine with 3GB heap.
 
-Cassandra will need at least 1GB of memory by itself if run on same machine.
+Grakn will need at least 1GB of memory by itself if run on same machine.
```

Comment:
These two paragraphs read like they're saying the same thing now. Maybe join them?